docs/source/utilities.rst

```diff
@@ -19,6 +19,8 @@ set_host_device_count
 ---------------------
 .. autofunction:: numpyro.util.set_host_device_count
 
+.. _inference_utilities:
```

Comment:
What does this do?

Reply:
I want to point to the docs of init strategies in AutoContinuous docs, so I create this label.

Reply:
I see. It would be nice to have initialization strategies within a separate header actually. Can we do that by using?

```
Initialization Strategies
^^^^^^^^^^^^^^^^^^^^^^^^
```

Then you could just link to that directly.

Reply:
Thanks! I just made some changes. Is it what you suggested? It looks good to me. :D

Reply:
Looks great!